--- a/configure.ac
+++ b/configure.ac
@@ -2711,6 +2711,11 @@ if test "$with_threads" = "yes" -o -z "$
 						# pthread (first!) on Linux
 fi
 
+if test $ac_sys_system = Darwin
+then
+  LIBS="$LIBS -framework CoreFoundation"
+fi
+
 # check if we need libintl for locale functions
 AC_CHECK_LIB(intl, textdomain,
 	[AC_DEFINE(WITH_LIBINTL, 1,
@@ -5163,11 +5168,6 @@ if test "x$ac_cv_file__dev_ptc" = xyes;
   [Define to 1 if you have the /dev/ptc device file.])
 fi
 
-if test $ac_sys_system = Darwin
-then
-	LIBS="$LIBS -framework CoreFoundation"
-fi
-
 AC_CACHE_CHECK([for %zd printf() format support], ac_cv_have_size_t_format, [dnl
 AC_RUN_IFELSE([AC_LANG_SOURCE([[
 #include <stdio.h>
